apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -363,7 +380,7 @@ def parse(element_html, data):
         data["submitted_answers"][name] = None
 
 
-def grade(element_html, data):
+def grade(element_html, data: pl.QuestionData) -> None:
```

Comment:
Missing a type annotation for element_html (this happens in a couple places in this file).

Reply:
Good catch, thanks.